### ISSUE TITLE
Deprecated lifecycle method warnings with React 16.9

### ISSUE THREAD

**Issue by benjaminMaille** [stadline/react-payline#2]
Hello,

I tried to update my project to React 16.9 but I have some warnings like :

```
react-dom.development.js:11494 Warning: componentWillMount has been renamed, and is not recommended for use. See https://fb.me/react-async-component-lifecycle-hooks for details.

* Move code with side effects to componentDidMount, and set initial state in the constructor.
* Rename componentWillMount to UNSAFE_componentWillMount to suppress this warning in non-strict mode. In React 17.x, only the UNSAFE_ name will work. To rename all deprecated lifecycles to their new names, you can run `npx react-codemod rename-unsafe-lifecycles` in your project source folder.

Please update the following components: SideEffect(NullComponent)
```

Seems to be coming from `"react-helmet": "^5.2.1"`

It is possible to upgrade this module ?

**Comment by alex-pex** [comment]
Seems related to https://github.com/stadline/react-payline/issues/1#issuecomment-670395221

**Comment by alex-pex** [comment]
This is fixed in `2.0.0-beta.0`
